src/plugin/resources/config.json

```diff
@@ -1,14 +1,22 @@
 {
   "coinform": {
     "apiUrl": "https://api.coinform.eu",
+    "logLevel": "info",
     "hideThreshold": 5,
     "categories": {
-      "accurate": "information",
-      "accurate_with_considerations": "information",
-      "unsubstantiated": "information",
-      "inaccurate_with_considerations": "misinformation",
-      "inaccurate": "misinformation",
-      "not_verifiable_post": "misinformation"
+      "credible": "label", 
+      "mostly_credible": "label", 
+      "credibility_uncertain": "label", 
+      "mostly_not_credible": "blur", 
+      "not_credible": "blur", 
+      "not_verifiable": "label"
+    },
+    "accuracy": {
```

Comment:
Is this structure useful anymore?

Reply:
Yes, until we will receive the label associated action (blur, ...) from the rule engine, through the API gateway, this structure is necessary for the plugin to classify the tweet/post with the appropriate action (to avoid it being hardcoded).

Reply:
Ok!